### PR TITLE
Fix to broken master

### PR DESCRIPTION
**Pull request by adrw**
master was broken with following error. Fixed by removing description key from frontmatter query.

Error:
```
There was an error in your GraphQL query:

- Unknown field 'description' on type 'MdxFrontmatter'.

File: ../packages/gatsby-theme-digital-garden/src/templates/note.js
```

### CODE REVIEW

File: packages/gatsby-theme-digital-garden/src/templates/note.js [hunk 1]
```diff
@@ -11,7 +11,6 @@ export const pageQuery = graphql`
       body
       frontmatter {
         title
-        description
```
Comment (adrw) [inline packages/gatsby-theme-digital-garden/src/templates/note.js]: Fixes broken master

File: packages/gatsby-theme-digital-garden/src/components/layout.js [hunk 2]
```diff
@@ -21,11 +21,7 @@ export default props => (
       })}
     />
     <Layout>
-      {'*' in props && (
```
Comment (adrw) [inline packages/gatsby-theme-digital-garden/src/components/layout.js]: Condition would never be met in latest Gatsby version meaning that SEO would never update on return to index after navigating the site. This now properly updates title and SEO tags when navigating back to home page.

**Comment by adrw** [comment]
@johno Release `0.1.0` is currently broken, this fixes it and will require another release. Thanks!

**Comment by ChristopherBiscardi** [comment]
@johno are we deprecating this repo in favor of gatsby-theme-notes?

**Comment by adrw** [comment]
@johno @ChristopherBiscardi If this repo isn't going to have continued active development, I'd be happy to accept ownership transfer and continue work on it. If this ends up being deprecated, I'll continue work on it in my fork. Thanks again for this great simple theme that you and the other contributors built! 👍

**Comment by johnletey** [comment]
Also, I'd be more than happy to help out @adrw continue developing/maintaining this theme if there ever is an ownership/maintainership transfer!

**Comment by johno** [comment]
> @johno are we deprecating this repo in favor of gatsby-theme-notes?

I plan on keeping this as a standalone set of themes to continue adding new features and functionality which we can move back into `gatsby-theme-blog` and `gatsby-theme-notes` when relevant for the larger ecosystem.

I would like to replace the internals of `gatsby-theme-digital-garden` by using `gatsby-theme-notes` as a parent theme.

**Comment by johno** [comment]
Thanks for the fix @adrw!

I'll add you and @johnletey as collaborators to the repo as well.

**Comment by adrw** [comment]
@johno Thanks for the merge and adding me as contributor!

**Comment by ChristopherBiscardi** [comment]
Dope, I love the idea of using this as a testing ground/child theme of gatsby-theme-notes. 👍

**Pull request merged**